package.json

```diff
@@ -11,7 +11,8 @@
     "lint:format": "prettier  --loglevel warn --write \"./**/*.{js,jsx,ts,tsx,css,md,json}\" ",
     "lint": "npm run lint:format && npm run lint:fix ",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "prepare": "husky install"
```

Comment:
Can we add this as an automated step?

https://typicode.github.io/husky/#/?id=manual

If not, let's include the instructions in a README so people know to run `prepare` after `npm install`

Reply:
Yes, Chuck. I will firstly revoke the PR,since I've found my fork lost. I will rebuild a new branch and create a new PR with an explanation about how to enable the Husky pre-commit check in the README.